### DISCUSSION TITLE
是否考虑加入gst？

### DISCUSSION THREAD

**Discussion by jerryuhoo** [PaddlePaddle/PaddleSpeech#1110] [Ideas]
在espnet中的fastspeech2模型中有加入gst，以合成不同语气风格的声音。是否考虑在fastspeech2模块中加入gst呢？

**Comment by yt605155624** [comment]
这个最近没时间搞了，不过应该和加入外挂的 spk emb 方法差不多吧，可以参考 aishell3/vc1 实例就是 fastspeech2 + ge2e，我理解 gst 是不是再搞一个 embedding 就行了

**Reply by jerryuhoo** [reply]
嗯，应该是的。如果有空的话我会尝试写写看。
